NameComparisonToolkit/Comparers/FirstLast.cs

```diff
@@ -9,7 +12,7 @@ public override bool Equals(Name x, Name y)
 
 	public override bool Contains(Name x, string y)
 		=> y.Contains(string.Join(" ", x.FirstName))
-		&& y.Contains(string.Join(" ", x.LastName));
+			&& y.Contains(string.Join(" ", x.LastName));
```

Comment:
why the extra tab/spacing?

Reply:
looks better, its not as dramatic in code as it seems here.